lib/foreman/gettext/support.rb

```diff
@@ -42,19 +42,31 @@ def self.add_text_domain(locale_domain, locale_dir)
           :report_warning => false
       end
 
-      def self.register_human_localenames
-        original_locale = FastGettext.locale
-        FastGettext.class.class_eval { attr_accessor :human_available_locales }
-        FastGettext.human_available_locales = []
-        FastGettext.default_available_locales.sort.each do |locale|
-          FastGettext.locale = locale
-          # TRANSLATORS: Provide locale name in native language (e.g. English, Deutsch or Portugues)
-          human_locale = _("locale_name")
-          human_locale = locale if human_locale == "locale_name"
-          FastGettext.human_available_locales << [human_locale, locale]
+      # Loading all available locales to get the language translation is slow
+      # (2 seconds for 20 languages), so let's lazy load that.
+      FastGettext.class_eval do
+        attr_accessor :mutex, :locales
```

Comment:
I don't think mutex and locales should be `attr_accessor`. At most `attr_reader`.

Also, why is `locales` there? We don't use it and it will be equal to `default_available_locales`. Shouldn't that just be a private variable?

Reply:
Ad mutex: it needs to be because I cannot initialize the mutex in the human_available_locales, it needs to be done during evaluation time. If you figure out better but working version, I will happily refactor it but I spent some time with this and this was the cleanest looking version of all. Hint: Rubocop will give you HARD time when you start modifying this: https://stackoverflow.com/questions/65922825/ruby-style-classvars-cop-confusion

Ad locales: this one can be changed to `@@locales` but it will look weird.

All and all, I think this is the cleanest version of all the 25 versions I tried...

Reply:
Just to save you some time - you can write it in a different way in Ruby, its just Rubocop would not let you commit this into our repo.

Reply:
Hmm you're right. Should we indicate that they are private by prefixing them with an underscore? That feels like a decent compromise.

I also think that we should name `locales` `human_locales` to indicate it's not just a list of locales. So
```suggestion
        attr_accessor :_mutex, :_human_locales
```